sympy/stats/joint_rv_types.py

```diff
@@ -80,20 +80,21 @@ class MultivariateNormalDistribution(JointDistribution):
 
     @property
     def set(self):
-        k = len(self.mu)
```

Comment:
did you test the constructor with a `list`?

Reply:
Yes, the test also use `list` moreover,
```python3
>>> from sympy.stats import *
>>> X = Normal('X', [0, 0], [[1, 0], [0, 1]])
>>> density(X)
MultivariateNormalDistribution(Matrix([
[0],
[0]]), Matrix([
[1, 0],
[0, 1]]))
```